Adafruit_NeoPixel.h

```diff
@@ -56,6 +56,7 @@ class Adafruit_NeoPixel {
     setPixelColor(uint16_t n, uint8_t r, uint8_t g, uint8_t b),
     setPixelColor(uint16_t n, uint32_t c),
     setBrightness(uint8_t),
+    setBrightness(uint8_t r, uint8_t g, uint8_t b),
     clear();
   uint8_t
    *getPixels(void) const,
```

Comment:
From http://stackoverflow.com/questions/2157458/using-const-in-classs-functions:

"Typically you want to declare all non-mutating non-static class methods as const. This allows calling code to use the const qualifier on pointers, and it helps catch mistakes."